fuse_variables/package.xml

```diff
@@ -12,16 +12,26 @@
   <author email="tmoore@locusrobotics.com">Tom Moore</author>
   <license>BSD</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>libceres-dev</depend>
-  <depend>fuse_core</depend>
-  <depend>pluginlib</depend>
-  <depend>roscpp</depend>
-  <test_depend>roslint</test_depend>
-  <test_depend>rostest</test_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+
+  <build_depend>libceres-dev</build_depend>
+  <build_depend>fuse_core</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_depend>rclcpp</build_depend>
+
+  <exec_depend>libceres-dev</exec_depend>
+  <exec_depend>fuse_core</exec_depend>
+  <exec_depend>pluginlib</exec_depend>
+  <exec_depend>rclcpp</exec_depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_cmake_pytest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
+  <test_depend>launch_pytest</test_depend>
```

Comment:
Also `<test_depend>launch</test_depend>`

Reply:
Doesn't it get transitively included along with `launch_pytest`? :o
https://github.com/ros2/launch/blob/rolling/launch_testing/package.xml

Reply:
Ah it does. I'm fine with not adding it then. I suggested it since `launch` is used directly: https://github.com/locusrobotics/fuse/pull/288/commits/7a247f8bd9753e0c185d8011cb6f48c537614452#diff-bb351725ed7327d13c0978085785de586768696ff7184ea559c96b3df64f0589R19-R20